hack/generate/samples/internal/ansible/advanced_molecule.go

```diff
@@ -0,0 +1,502 @@
+	_, err = ma.ctx.Run(cmd)
+	pkg.CheckError("adding plugins/", err)
+
+	log.Infof("adding fixture_collection/")
+	cmd = exec.Command("cp", "-r", filepath.Join("../../../hack/generate/samples/internal/ansible/testdata/fixture_collection/"), filepath.Join(ma.ctx.Dir, "fixture_collection/"))
+	_, err = ma.ctx.Run(cmd)
+	pkg.CheckError("adding fixture_collection/", err)
+
+	log.Infof("replacing watches.yaml")
+	cmd = exec.Command("cp", "-r", filepath.Join("../../../hack/generate/samples/internal/ansible/testdata/watches.yaml"), ma.ctx.Dir)
+	_, err = ma.ctx.Run(cmd)
+	pkg.CheckError("replacing watches.yaml", err)
+
+	log.Infof("adding tasks/")
+	cmd = exec.Command("cp", "-r", filepath.Join("../../../hack/generate/samples/internal/ansible/testdata/tasks/"), filepath.Join(ma.ctx.Dir, "molecule/default/"))
+	_, err = ma.ctx.Run(cmd)
+	pkg.CheckError("adding tasks/", err)
+
+	log.Infof("adding secret playbook")
+	cmd = exec.Command("cp", "-r", filepath.Join("../../../hack/generate/samples/internal/ansible/testdata/secret.yml"), filepath.Join(ma.ctx.Dir, "playbooks/secret.yml"))
+	_, err = ma.ctx.Run(cmd)
+	pkg.CheckError("adding secret playbook", err)
+
+	log.Infof("adding inventory/")
+	cmd = exec.Command("cp", "-r", filepath.Join("../../../hack/generate/samples/internal/ansible/testdata/inventory/"), filepath.Join(ma.ctx.Dir, "inventory/"))
+	_, err = ma.ctx.Run(cmd)
+	pkg.CheckError("adding inventory/", err)
+}
+
+func (ma *AdvancedMolecule) updateDockerfile() {
+	log.Infof("replacing project Dockerfile to use ansible base image with the dev tag")
+	err := testutils.ReplaceRegexInFile(filepath.Join(ma.ctx.Dir, "Dockerfile"), "quay.io/operator-framework/ansible-operator:.*", "quay.io/operator-framework/ansible-operator:dev")
+	pkg.CheckError("replacing Dockerfile", err)
+
+	log.Infof("inserting code to Dockerfile")
+	err = kbtestutils.InsertCode(filepath.Join(ma.ctx.Dir, "Dockerfile"), "COPY playbooks/ ${HOME}/playbooks/", dockerfileFragment)
+	pkg.CheckError("replacing Dockerfile", err)
+}
+
+func (ma *AdvancedMolecule) updatePlaybooks() {
+	log.Infof("adding playbook for argstest")
+	err := testutils.ReplaceInFile(filepath.Join(ma.ctx.Dir, "playbooks", "argstest.yml"),
+		originalPlaybookFragment, argsPlaybook)
+	pkg.CheckError("adding playbook for argstest", err)
+
+	log.Infof("adding playbook for casetest")
+	err = testutils.ReplaceInFile(filepath.Join(ma.ctx.Dir, "playbooks", "casetest.yml"),
+		originalPlaybookFragment, casePlaybook)
+	pkg.CheckError("adding playbook for casetest", err)
+
+	log.Infof("adding playbook for inventorytest")
+	err = testutils.ReplaceInFile(filepath.Join(ma.ctx.Dir, "playbooks", "inventorytest.yml"),
+		"---\n- hosts: localhost\n  gather_facts: no\n  collections:\n    - community.kubernetes\n    - operator_sdk.util\n  tasks:\n    - import_role:\n        name: \"inventorytest\"",
```

Comment:
I really don't like the contents having to be specified in this fashion. I've done this in the past and this always causes annoying formatting issues.

Reply:
I use an IDE so  I can just copy and past and it fixes the string with the characters. But I see what do you mean. I will change its style to be more readable and be easier to keep maintained.